doc/tutorials/introduction_to_ttl/introduction_to_ttl.md

```diff
@@ -5,36 +5,34 @@ CONTENTS:
 ---------
 
 - [Introduction to the Tensor \& Tiling Library (TTL)](#introduction-to-the-tensor--tiling-library-ttl)
-  - [CONTENTS:](#contents)
-  - [Introduction](#introduction)
-    - [Example](#example)
-  - [Background](#background)
-    - [Tiling](#tiling)
-    - [Design Principles](#design-principles)
-  - [TTL Logical Tiling](#ttl-logical-tiling)
-    - [TTL\_create\_shape](#ttl_create_shape)
-    - [TTL\_create\_overlap](#ttl_create_overlap)
-    - [TTL\_tile\_t](#ttl_tile_t)
-    - [TTL\_tiler\_t](#ttl_tiler_t)
-  - [TTL Physical Tensors](#ttl-physical-tensors)
-    - [TTL\_create\_layout](#ttl_create_layout)
-    - [TTL\_\[const\]\_\[int,ext\]\_\[sub\]\_tensor\_t](#ttl_const_intext_sub_tensor_t)
-    - [TTL\_io\_tensors](#ttl_io_tensors)
-  - [TTL Import and Export Transactions](#ttl-import-and-export-transactions)
-  - [TTL Pipelining Schemes](#ttl-pipelining-schemes)
-    - [Pipelining Iterations](#pipelining-iterations)
-    - [Duplex Buffering](#duplex-buffering)
-    - [Double Buffering](#double-buffering)
-    - [Simplex Buffering](#simplex-buffering)
-  - [TTL Tiling Loop Parallelization](#ttl-tiling-loop-parallelization)
-  - [Debugging](#debugging)
-  - [Tiling Code Examples](#tiling-code-examples)
-    - [Duplex Buffering Scheme](#duplex-buffering-scheme)
-    - [Double Buffering Scheme](#double-buffering-scheme)
```

Comment:
Double buffering scheme removed?

Reply:
Killed the reformatting as well. I had a regen TOC in a Vscode plugin.